### PR TITLE
DFE-827 - Publication page analytics event on see previous releases

### PR DESCRIPTION
**Pull request by HiveIT-Lewis**

### CODE REVIEW

File: src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx [hunk 1]
```diff
@@ -145,7 +146,17 @@ class PublicationReleasePage extends Component<Props> {
                   <strong>{data.releaseName}</strong>
                 </dd>
                 <dd>
-                  <Details summary={`See previous ${releaseCount} releases`}>
+                  <Details
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx]: Looks good to me, if you can find out why the build failed?

Comment (HiveIT-Lewis) [reply src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx]: Looks like I just need to remove a duplicate.

`> explore-education-statistics-frontend@0.1.0 tsc /home/vsts/work/1/s/src/explore-education-statistics-frontend
> tsc --noEmit

src/modules/find-statistics/PublicationReleasePage.tsx(13,10): error TS2300: Duplicate identifier 'logEvent'.
src/modules/find-statistics/PublicationReleasePage.tsx(15,10): error TS2300: Duplicate identifier 'logEvent'.`

**Comment by bennettstuart** [comment]
Magic™ ✨

File: src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx [hunk 2]
```diff
@@ -145,7 +146,17 @@ class PublicationReleasePage extends Component<Props> {
...
+                    summary={`See previous ${releaseCount} releases`}
+                    onToggle={(open: boolean) =>
+                      open &&
+                      logEvent(
+                        window.location.pathname,
+                        'Open previous releases accordion',
+                        'Previous Releases',
```
Comment (bennettstuart) [inline src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx]: Let's get event properties consistent before merging

**Pull request merged**